packages/markdown-it-myst/src/directives.ts

```diff
@@ -171,44 +171,44 @@ function directiveArgToTokens(arg: string, lineNumber: number, state: StateCore)
   return nestedPartToTokens('directive_arg', arg, lineNumber, state, 'run_directives', true);
 }
 
-function simplifyDirectiveOptions(options?: [string, string][]) {
+function getDirectiveOptions(options?: [string, string | true][]) {
   if (!options) return undefined;
-  const simplified: Record<string, string | boolean | number> = {};
+  const simplified: Record<string, string | true> = {};
   options.forEach(([key, val]) => {
     if (simplified[key] !== undefined) {
       return;
-    } else if (!isNaN(Number(val))) {
-      simplified[key] = Number(val);
-    } else if (typeof val === 'string' && val.toLowerCase() === 'true') {
-      simplified[key] = true;
-    } else if (typeof val === 'string' && val.toLowerCase() === 'false') {
-      simplified[key] = false;
-    } else {
-      simplified[key] = val;
```

Comment:
I am moving this logic to `myst-parser` and this is just passing either a `true` or the actual content.

Reply:
Ah, so this stuff isn't really actually needed at all - it is redundant to the coercion in `myst-parser` (which you've fixed up for the `true/'true'` change).